pandas/_libs/tslibs/offsets.pyx

```diff
@@ -3627,24 +3629,95 @@ cdef class CustomBusinessHour(BusinessHour):
         Normalize start/end dates to midnight before generating date range.
     weekmask : str, Default 'Mon Tue Wed Thu Fri'
         Weekmask of valid business days, passed to ``numpy.busdaycalendar``.
+    holidays : list
+        List/array of dates to exclude from the set of valid business days,
+        passed to ``numpy.busdaycalendar``.
+    calendar : np.busdaycalendar
+        Calendar to integrate.
     start : str, time, or list of str/time, default "09:00"
         Start time of your custom business hour in 24h format.
     end : str, time, or list of str/time, default: "17:00"
         End time of your custom business hour in 24h format.
+    offset : timedelta, default timedelta(0)
+        Time offset to apply.
```

Comment:
have you noticed whether this parameter has any effect, or is it unused?

Reply:
I haven’t checked it yet, but I’ll do it. If the parameter has any effect, I’ll add an example.

Reply:
My checks show that the parameter `offset` hasn't any effect on how the `CustomBusinessHour` works. That is why I don’t mark it as a parameter of `CustomBusinessHour`.